spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,output:tier::int,output:scalerank::int
+national_park;park or protected land;battlefield,!United States Forest Service,>=3,>=300000000,1,3
```

Comment:
To make this work better outside of United States the filter needs to be expanded to include kinds of `protected_area ` and several `protected_class` values:

```
                    all:
                        - kind: [national_park, battlefield, protected_area]
                    any:
                        - not: { operator: [ "United States Forest Service" ] }
                        - protect_class: ['2','3','5']
```